builtin/providers/digitalocean/resource_digitalocean_droplet_test.go

```diff
@@ -46,6 +46,31 @@ func TestAccDigitalOceanDroplet_Basic(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanDroplet_WithID(t *testing.T) {
+	var droplet godo.Droplet
+	rInt := acctest.RandInt()
+	// TODO: not hardcode this as it will change over time
+	centosID := 22995941
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDigitalOceanDropletDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckDigitalOceanDropletConfig_withID(centosID, rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDigitalOceanDropletExists("digitalocean_droplet.foobar", &droplet),
+				),
+				Destroy: false,
+			},
+			{
+				Config:   testAccCheckDigitalOceanDropletConfig_withID(centosID, rInt),
+				PlanOnly: true,
+			},
```

Comment:
We actually refresh the state as part of every test step, so this is not necessary - would you mind removing the second step + `Destroy: false`?